src/Fable.Cli/Main.fs

```diff
@@ -193,7 +193,7 @@ module private Util =
             do! BabelPrinter.run writer mapGen babel
             do! mapPrinter outPath
 
-            Log.always("Compiled " + File.getRelativePathFromCwd com.CurrentFile)
+            logger <| "Compiled " + File.getRelativePathFromCwd com.CurrentFile
```

Comment:
Consider

```fsharp
logger $"Compiled {File.getRelativePathFromCwd com.CurrentFile}"
```

instead?

Reply:
Yes, I have a tendency to just use concatenation in simple cases because formatting/interpolation is more expensive. But shouldn't really matter in this case :)